Description: Fix to build with wxWidgets 3.0
 Should also work with wxWidgets 2.8.
Author: Olly Betts <olly@survex.com>
Last-Update: 2014-03-12

Index: kicad-0.20141025+bzr4029/kicad/kicad.h
===================================================================
--- kicad-0.20141025+bzr4029.orig/kicad/kicad.h
+++ kicad-0.20141025+bzr4029/kicad/kicad.h
@@ -44,8 +44,10 @@
 // With a recent wxWidget, we can use the wxFileSystemWatcherEvent
 // to monitor files add/remove/rename in tree project
 #if wxCHECK_VERSION( 2, 9, 4  )
+#if wxUSE_FSWATCHER
 #define KICAD_USE_FILES_WATCHER
 #endif
+#endif
 
 extern const wxString g_KicadPrjFilenameExtension;
 
